### ISSUE TITLE
[BUG REPORT] SNR estimator broken?

### ISSUE THREAD

**Issue by lacan** [aarpon/hrm#21]
**Describe the bug**
Due to how long the SNR estimator runs now, we had stopped using it. I tried it with a single channel dataset and have much trouble interpreting what it written...  It just says Original 3 times and no links to help are displayed

**To Reproduce**
Request an SNR estimation from a single channel image. I do not know the behaviour for multichannel images as of this writing

**Expected behavior**
The classic Estimated SNR value, a lowest estimation and a highest estimation. Or something like on the documentation here
https://svi.nl/HuygensRemoteManagerHelpSnrEstimator

**Screenshots**
![image](https://github.com/aarpon/hrm/assets/319932/03af71f5-7dd5-4e44-a5e2-3ca0e8addd16)

**Environment**
***Server-side***
Huygens Remote Manager |  
-- | --
HRM version | 3.9
HRM required database version | 20
HRM current database version | 20
HuCore minimum required version | 21.10.1
HuCore current version | 23.4.0-p6
HuCore server type | medium
Licenses in use |  
Detailed license information |  
HuCore license file | valid
HRM-OMERO Connector |  
Version |  
System |  
Operating system | Linux x86_64
Kernel release | 3.10.0-1160.99.1.el7.x86_64
LAMP versions |  
Apache version | 2.4.6
Database type and version | mysqli 5.7.43
PHP (Apache mod) version | 8.2.11
Configuration |  
Send test e-mail to HRM admin |  
Memory limit | 128MB
Max execution time | 30s
File downloads |  
HRM configuration | disabled
Maximum post size |  
php.ini | 8MB
HRM configuration | limited by php.ini.
in use | 8MB
File uploads |  
php.ini | enabled
HRM configuration | enabled
in use | enabled
Maximum upload file size |  
php.ini | 2MB
HRM configuration | limited by php.ini.
in use | 2MB

***Client-side***
FireFox 118.0.1 (64-bit)

**Comment by KevinNamink** [comment]
I agree it is a bit slow for larger images.

I've tried a few images but I cannot reproduce this undecipherable output.
What type of image are you using? 
What are the image dimensions?

**Comment by lacan** [comment]
Dear @KevinNamink, 

Thanks for looking into it. It is a lif file with 2048x2048x231 z slices

https://drive.switch.ch/index.php/s/Wip2HmXnHoxppg2
Password is hrm and the link will expire on friday. 

Best
Oli

**Comment by KevinNamink** [comment]
Thanks for the file. I can now reproduce it.
I've found and fixed the issue. The cause was in Huygens. In Huygens 23.10 this particular issue will be fixed. 

I've also found a bug with the "legacy" SNR estimation mode. That should also work again with Huygens 23.10. This might be a faster method if that is important.

I think these problems are specific to Huygens 23.04, I'll try to put these fixes in a patch for that version as well. (I cannot 100% guarantee this patch will come.)
